service/src/test/java/uk/gov/hmcts/reform/fpl/utils/providers/NotCloseableGeneratedCareOrderProvider.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.fpl.utils.providers;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+
+import java.util.stream.Stream;
+
+import static uk.gov.hmcts.reform.fpl.enums.GeneratedOrderSubtype.INTERIM;
+import static uk.gov.hmcts.reform.fpl.enums.GeneratedOrderType.BLANK_ORDER;
+import static uk.gov.hmcts.reform.fpl.enums.GeneratedOrderType.CARE_ORDER;
+import static uk.gov.hmcts.reform.fpl.enums.GeneratedOrderType.SUPERVISION_ORDER;
+
+public class NotCloseableGeneratedCareOrderProvider implements ArgumentsProvider {
```

Comment:
Couldn't we make those provider nested classes? Not sure if I see massive benefit of having them external.